docs/maintenance-plan.md

```diff
@@ -0,0 +1,189 @@
+Notes:
+
+### Nginx Ingress Controller
+
+Update interval: Monthly
+Release URL: <https://github.com/kubernetes/ingress-nginx/releases>
+Upgrade Docs URL: <https://kubernetes.github.io/ingress-nginx/deploy/upgrade/#with-helm>
+Expect downtime: Highly likely
+Time to do: Unkonwn
+Runbook: [Upgrading Nginx-ingresss](./runbooks/upgrading-support-workloads#ingress-nginx)
+Notes:
+
+### K8up
+
+DO NOT UPGRADE.
+From time to time we should checkin with this page [K8Up in lagoon](https://docs.lagoon.sh/installing-lagoon/lagoon-backups/#lagoon-backups)
+, where they'll hopefully give and update when it is possible to update K8Up
+to a later version than version 1.x.x
+
+### Loki
+
+Update interval: somewhat monthly
+Release URL: <https://artifacthub.io/packages/helm/grafana/loki>
+Upgrade Docs URL: <https://grafana.com/docs/loki/latest/setup/upgrade/>
+Expect downtime: Might be some, but nothing that will hit the libraries
+Time to do: Unkown
+Runbook: [Upgrade Loki](./runbooks/upgrading-support-workloads.md#loki)
+Notes:
+
+### Minio
+
+As we're replacing Minio, we will have to do a section on what ever tool
+  we're replacing it with.
+
+### Prometheus
+
+Update interval: Weekly
+Release URL: <https://github.com/prometheus-community/helm-charts/releases?q=kube-prometheus-stack&expanded=true>
+Upgrade Docs URL: <https://artifacthub.io/packages/helm/prometheus-community/kube-prometheus-stack#upgrading-chart>
+Expect downtime: Probably, but nothing the libraries will be affected by
+Time to do: Unkown
+Runbook: [Upgrading Prometheus](./runbooks/upgrading-support-workloads.md#prometheus)
+Notes:
+
+### Promtail
+
+Update interval: monthly-quarterly
+Release URL: <https://github.com/grafana/helm-charts/releases?q=promtail&expanded=true>
+Upgrade Docs URL: <https://github.com/grafana/helm-charts/blob/main/charts/promtail/README.md#upgrading>
+Expect downtime: Probably, but nothing that concerns the libraries.
+Time to do: Unknown
+Runbook: [Upgrading Promtail](./runbooks/upgrading-support-workloads.md#upgrade-promtail)
+Notes:
+
+## DPL Shell parts
+
+The DPL Shell integrates our day to day tools. The ones, that are not upgraded
+automatically when a new version of DPL Shell is created, are listed below.
+Most of these, if not all, are watched by dependabot on GitHub, so we
+are notified about updates pretty quickly.
+
+### Terraform
+
+Update interval: Monthly
+Release URL: <https://hub.docker.com/r/hashicorp/terraform/tags?page=1&ordering=last_updated>
+Expect downtime: None
+Time to do: 1 hour
+Runbook:
+Notes:
+
+### Azure CLI
+
+Update interval: Unknown
+Release URL: <https://mcr.microsoft.com/v2/azure-cli/tags/list>
+Expect downtime: None
+Time to do: 1 hour
+Runbook:
+Notes:
+
+### kubectl
+
+Update interval: Quarterly
+Release URL: <https://github.com/Azure/kubelogin/releases>
+Expect downtime: None
+Time to do: 1 hour
+Runbook:
+Notes:
+
+### Krew
+
+Update interval: Unkown, hasen't been updated since 2023
```

Comment:
Makes sense to establish it. Suggestion: 6 months.

Reply:
:+1: